vmware_aria_operations_integration_sdk/mp_init.py

```diff
@@ -345,6 +352,19 @@ def main() -> None:
             "For more information visit https://vmware.github.io/vmware-aria-operations-integration-sdk/get_started/#template-projects",
         )
 
+        if language == "java":
+            package_name = prompt(
+                "Enter package name: ",
+                default="com.mycompany",
+                validator=ChainValidator(
+                    [
+                        NotEmptyValidator("Java Package"),
+                        NoUppercaseValidator("Java Package"),
+                    ]
+                ),
```

Comment:
There's nothing wrong with this, but what we've done in the past for this case is to have validators inherit from `NotEmptyValidator` - for example, `NewProjectDirectoryValidator` or `UniquenessValidator`